src/iexamine.cpp

```diff
@@ -1971,8 +1971,13 @@ void iexamine::kiln_empty(player &p, const tripoint &examp)
     if( !p.has_charges( "fire" , 1 ) ) {
         add_msg( _("This kiln is ready to be fired, but you have no fire source.") );
         return;
-    } else if( !query_yn( _("Fire the kiln?") ) ) {
-        return;
+    } else {
+        add_msg( _( "This kiln contains %sL of material, and is ready to be fired." ),
```

Comment:
This will show volume strictly in liters, while players might be using other volume units.

Reply:
fixed.